swap_meet/decor.py

```diff
@@ -1,2 +1,24 @@
-class Decor:
-    pass
\ No newline at end of file
+from .item import Item
+
+class Decor(Item):
+    '''
+    A sub-class of Item, indicating a type of item a vendor might have.
+    
+    Attributes:
+        condition (float): optional descr. of item condition, 0 if not defined
+        category (str): description of item category, always set to "Decor"
+    ''' 
+
+    def __init__(self, condition=0):
+        '''
+        Parameter: condition (float): optional description of item 
+            condition, 0 if not defined
+        '''
+
+        self.category = "Decor"
+        self.condition = condition
```

Comment:
Great! Another option that works is calling the constructor of the parent (Item) class like `super().__init__(condition=condition, category='Decor')`